docs/guide.mdx

```diff
@@ -119,12 +119,38 @@ it'll work just fine.
 
 ### Languages & Frameworks
 
-Sapphire is programmable using any language that targets the EVM, such as Solidity
-or Vyper. If you prefer to use an Ethereum framework like Hardhat or Foundry, you
-can also use those with Sapphire; all you need to do is set your Web3 gateway URL.
+Sapphire is programmable using any language that targets the EVM, such as Solidity,
```

Comment:
Let's move the *Wallets* and *Languages & Frameworks* under the *Integrating Sapphire* section.

Reply:
This has been removed from the guide, it's addressed in the wagmi README